lesson_07/objects/objects_app/src/test/java/com/codedifferently/lesson7/Jordan_Eldridge/AnimalTest.java

```diff
@@ -1,40 +1,69 @@
-package com.codedifferently.lesson7.Jordan_Eldridge;
+package com.codedifferently.lesson7.jordan_eldridge;
 
 import static org.junit.jupiter.api.Assertions.*;
 
-import com.codedifferently.lesson7.Jordan_Eldridge.Animal.AnimalType;
+import com.codedifferently.lesson7.jordan_eldridge.Animal.AnimalType;
+import com.codedifferently.lesson7.jordan_eldridge.Animal.InvalidAgeException;
+import java.util.List;
 import org.junit.jupiter.api.Test;
 
 public class AnimalTest {
 
   @Test
-  public void testCanBreathe() {
+  public void testConstructor() {
     Animal animal = new Animal("Dog", true, 5, "Brown", "Bark", AnimalType.MAMMAL);
-    assertTrue(animal.isCanBreathe());
+    assertEquals("Dog", animal.getName());
+    assertTrue(animal.getCanBreathe());
+    assertEquals(5, animal.getAge());
+    assertEquals("Brown", animal.getColor());
+    assertEquals(AnimalType.MAMMAL, animal.getType());
   }
 
   @Test
-  public void testGetAge() {
-    Animal animal = new Animal("Dog", true, 5, "Brown", "Bark", AnimalType.MAMMAL);
-    assertEquals(5, animal.getAge());
+  public void testCanFly() {
+    Animal animal = new Animal("Owl", true, 1, "Brown", "whistles", AnimalType.BIRD);
+    assertEquals("This animal can fly!", animal.canFly());
   }
 
   @Test
-  public void testGetColor() {
-    Animal animal = new Animal("Dog", true, 5, "Brown", "Bark", AnimalType.MAMMAL);
-    assertEquals("Brown", animal.getColor());
+  public void testMakeSound() {
+    Animal animal = new Animal("lizard", true, 3, "Green", "Hissing", AnimalType.REPTILE);
+    animal.makeSound();
   }
 
   @Test
-  public void testGetType() {
-    Animal animal = new Animal("Dog", true, 5, "Brown", "Bark", AnimalType.MAMMAL);
-    assertEquals(AnimalType.MAMMAL, animal.getType());
+  public void testRepeatSound() {
+    Animal snake = new Animal("henry", true, 2, "brown", "hissing", AnimalType.REPTILE);
+    snake.repeatSound(2);
   }
 
   @Test
-  public void testSetName() {
+  public void testSetAge() {
     Animal animal = new Animal("Dog", true, 5, "Brown", "Bark", AnimalType.MAMMAL);
-    animal.setName("Cat");
-    assertEquals("Cat", animal.getName());
+    Exception exception =
+        assertThrows(
```

Comment:
nit: You can also use `assertThatThrowsBy` to check assertions, which is how the cool kids do it nowadays I believe.

Reply:
I could not find out where this was at !

Reply:
I misspelled it (should be `assertThatThrownBy`), but a Google search leads me to this page. You should be able to import and use it. You can also see example usage in PR #203. 

https://www.javadoc.io/doc/org.assertj/assertj-core/3.8.0/org/assertj/core/api/Assertions.html